docs/operation-guide.md

```diff
@@ -73,14 +73,18 @@ On GKE, local SSD volumes by default are limited to 375 GiB size and perform wor
 
 For proper performance, you must:
 
-* install the Linux guest environment, which can only be done on the Ubuntu image, not the COS image
+* install the Linux guest environment on the Ubuntu image or use a recent COS image
 * make sure SSD is mounted with the `nobarrier` option.
 
-We have a [daemonset which does the above performance fixes](../manifests/gke/local-ssd-optimize.yaml).
-We also have a [daemonset that fixes performance and combines all SSD disks together with lvm](../manifests/gke/local-ssd-provision.yaml).
+We also have a [daemonset](../manifests/gke/local-ssd-provision.yaml) that
+* fixes any performance issues
+* remounts local SSD disks with a UUID for safety
+* On Ubuntu combines all local SSD disks into one large disk with lvm tools.
```

Comment:
Should we let the user decide whether to combine the disks? In case users need to deploy multiple TiKV instances on one node.

Reply:
In the cloud I don't think it makes sense to deploy multiple TiKV instances to one node. I am more concerned about a cluster that both runs tidb-operator and something else. But it is up to them to take these daemonsets and use them appropriately.